lib/rom/factory/dsl.rb

```diff
@@ -88,28 +89,36 @@ def timestamps
       #
       #   @param [Symbol] type The value type to generate
       #
-      # @overload fake(api, type)
+      # @overload fake(genre, type)
       #   @example
       #     f.email { fake(:internet, :email) }
       #
-      #   @param [Symbol] api The faker API identifier ie. :internet, :product etc.
+      #   @param [Symbol] genre The faker API identifier ie. :internet, :product etc.
       #   @param [Symbol] type The value type to generate
       #
-      # @overload fake(api, type, *args)
+      # @overload fake(genre, type, **options)
       #   @example
-      #     f.email { fake(:number, :between, 10, 100) }
+      #     f.email { fake(:number, :between, from: 10, to: 100) }
       #
-      #   @param [Symbol] api The faker API identifier ie. :internet, :product etc.
+      #   @param [Symbol] genre The faker API identifier ie. :internet, :product etc.
       #   @param [Symbol] type The value type to generate
-      #   @param [Array] args Additional arguments
+      #   @param [Hash] options Additional arguments
+      #
+      # @overload fake(genre, subgenre, type, **options)
+      #   @example
+      #     f.quote { fake(:books, :dune, :quote, character: 'stilgar') }
+      #
+      #   @param [Symbol] genre The Faker genre of API i.e. :books, :creature, :games etc
+      #   @param [Symbol] subgenre The subgenre of API i.e. :dune, :bird, :myst etc
+      #   @param [Symbol] type the value type to generate
+      #   @param [Hash] options Additional arguments
```

Comment:
Worth calling out that Faker v2 added additional hierarchy so `api` was insufficient. `genre`/`subgenre` is what I came up with.